homework14/js/main.js

```diff
@@ -0,0 +1,154 @@
+function fieldNotEmpty(field){
+    if(field === '' || field === ' '){
+        alert('Field not be empty!');
+    }
+}
+
+//Return the century of the input year. The input will always be a 4 digit string, so there is no need for validation.
+const dateInput = document.querySelector('.first-block__input');
+const centuryButton = document.querySelector('.first-block__button');
+const resultCentury = document.querySelector('.first-block__result');
+
+function showCentury(stringOfYear){
+    if(stringOfYear === '' || stringOfYear === ' '){
+        alert('Field not be empty!');
+    } else
+    if(stringOfYear !== 4){
+        alert('Enter a year greater than 999')
+    } else
+    if(stringOfYear < 1100){
+        resultCentury.innerHTML = `It's 11th`;
+    } else if(stringOfYear < 1200){
+        resultCentury.innerHTML = `It's 12th`;
+    } else if(stringOfYear < 1300){
+        resultCentury.innerHTML = `It's 13th`;
+    } else if(stringOfYear < 1400){
+        resultCentury.innerHTML = `It's 14th`;
+    } else if(stringOfYear < 1500){
+        resultCentury.innerHTML = `It's 15th`;
+    } else if(stringOfYear < 1600){
+        resultCentury.innerHTML = `It's 16th`;
+    } else if(stringOfYear < 1700){
+        resultCentury.innerHTML = `It's 17th`;
+    } else if(stringOfYear < 1800){
+        resultCentury.innerHTML = `It's 18th`;
+    } else if(stringOfYear < 1900){
+        resultCentury.innerHTML = `It's 19th`;
+    } else if(stringOfYear < 2000){
+        resultCentury.innerHTML = `It's 20th`;
+    } else if(stringOfYear < 2100){
+        resultCentury.innerHTML = `It's 21th`;
+    }
+}
```

Comment:
😮

Why so hardcoded?

Reply:
This has to be refactored. That code is hard to maintain and a lot of repeatings!